.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+AllCops:
+  TargetRubyVersion: 2.3
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
you can use our stylecheck for rubocop: https://rubygems.org/gems/ragnarson-stylecheck